tools/build_adafruit_bins.sh

```diff
@@ -7,6 +7,7 @@ rm -rf ports/nrf/build*
 # Alphabetical.
 HW_BOARDS="\
 arduino_zero \
+arduino_mkr1300 \
```

Comment:
These are alphabetized. Could you `move arduino_mkr1300` up one line?

Reply:
Oh of course